<!-- -*- mode: XML -*- -->
<launch>
  <rosparam file="$(find dynamixel_controller)/param/mimi_specification.yaml" command="load" ns="mimi_specification" />
  <node pkg="dynamixel_controller" name="head_motor_ctrl" type="head_motor_ctrl.py" output="screen" />
</launch>
